tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -478,6 +478,12 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
                     get_jax_decoder(args['input_model'], args)
                 else:
                     raise Error("JAX Frontend is not available.")
+            if model_framework == "tf" and "nncf" in sys.modules:
+                try:
+                    from nncf.tensorflow.strip import strip as nncf_tf_strip
+                    args['input_model'] = nncf_tf_strip(args['input_model'])
```

Comment:
Shouldn't we additionally check that `args['input_model']` is of `tf.keras.Model` type just avoid any mistakes in the transformation?

Reply:
We have already included that check inside the `strip()` method. Therefore, if the model is not a `tf.keras.Model`, the method does nothing.